src/ImageProcessor.Web.Plugins.AzureBlobCache/AzureBlobCache.cs

```diff
@@ -548,8 +548,7 @@ private static void TrySetIfModifiedSinceDate(HttpContext context, HttpWebReques
                 }
                 else
                 {
-                    ImageProcessorBootstrapper.Instance.Logger.Log<string>("Unable to parse date " +
-                                                                      context.Request.Headers["If-Modified-Since"]);
+                    ImageProcessorBootstrapper.Instance.Logger.Log<string>("Unable to parse date " + context.Request.Headers["If-Modified-Since"] + " for " + context.Request.Url, "ImageProcessor.Web.Plugins.AzureBlobCache.AzureBlobCache");
```

Comment:
So close! The logger works just like the Umbraco one.

```
ImageProcessorBootstrapper.Instance.Logger.Log<AzureBlobCache>("Unable to parse date " + context.Request.Headers["If-Modified-Since"] + " for " + context.Request.Url);
```